backend/src/main/java/gov/cdc/usds/simplereport/config/GraphQlConfig.java

```diff
@@ -46,8 +52,12 @@ public DataFetcherExceptionResolver dataFetcherExceptionResolver() {
 
       log.error("Will replace the following exception with a GenericGraphqlException: ", exception);
 
+      if (exception instanceof ResourceException) {
```

Comment:
Unfortunately, Okta uses a `ResourceException` for basically everything that goes wrong on their side 😅 So you might need to be more specific about which kinds of errors we're setting the duplicateUserMessage on, or this error message could be shown to users anytime Okta has a hiccup. 

As in the comment below, my preference here would be to catch and re-throw the exception at the source, rather than changing it in this configuration file - I think you'll need to do it in both the [`createUser`](https://github.com/CDCgov/prime-simplereport/blob/89a5a7bbe4a5c6442fe7cb32e0bce97b61e0f8be/backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/LiveOktaRepository.java#L193-L193) and [`updateUser`](https://github.com/CDCgov/prime-simplereport/blob/89a5a7bbe4a5c6442fe7cb32e0bce97b61e0f8be/backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/LiveOktaRepository.java#L247-L247) methods.
I'm happy to pair on this if you'd like, testing Okta exceptions can be tricky 😄 😭

Reply:
Just checked our Azure logs - the specific exception will probably be 
`com.okta.sdk.resource.ResourceException: HTTP 400, Okta E0000001 (Api validation failed: login - login: An object with this field already exists in the current organization)`

Reply:
Ohhh! that is good to know! Thank you -- I'll try that out!! :D